javascript/builtins/WebAssembly.json

```diff
@@ -0,0 +1,1350 @@
+{
+  "javascript": {
+    "builtins": {
+      "WebAssembly": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly",
+          "support": {
+            "webview_android": {
+              "version_added": "57"
+            },
+            "chrome": {
+              "version_added": "57"
+            },
+            "chrome_android": {
+              "version_added": "57"
+            },
+            "edge": {
+              "version_added": "16"
```

Comment:
I've got this listed as being available in 15 behind a flag. Where did you get 16 from? I did quite a bit of searching, and couldn't find any such info ;-)

Reply:
I saw https://twitter.com/MSEdgeDev/status/894607665334177792 and caniuse is saying 16 now, too.